exasol_integration_test_docker_environment/lib/api/common.py

```diff
@@ -1,145 +1,13 @@
-    temporary_base_directory: Optional[str],
-    cache_directory: Optional[str],
-    build_name: Optional[str],
-):
-    luigi.configuration.get_config().set(
-        "build_config", "force_rebuild", str(force_rebuild)
-    )
-    luigi.configuration.get_config().set(
-        "build_config", "force_rebuild_from", json.dumps(force_rebuild_from)
-    )
-    luigi.configuration.get_config().set("build_config", "force_pull", str(force_pull))
-    set_output_directory(output_directory)
-    if temporary_base_directory is not None:
-        luigi.configuration.get_config().set(
-            "build_config", "temporary_base_directory", temporary_base_directory
-        )
-    if cache_directory is not None:
-        luigi.configuration.get_config().set(
-            "build_config", "cache_directory", cache_directory
-        )
-    if build_name is not None:
-        luigi.configuration.get_config().set("build_config", "build_name", build_name)
-    luigi.configuration.get_config().set(
-        "build_config", "log_build_context_content", str(log_build_context_content)
-    )
-
-
-def set_output_directory(output_directory):
-    if output_directory is not None:
-        luigi.configuration.get_config().set(
-            "build_config", "output_directory", output_directory
-        )
-
-
-def set_docker_repository_config(
-    docker_password: Optional[str],
-    docker_repository_name: Optional[str],
-    docker_username: Optional[str],
-    tag_prefix: str,
-    kind: str,
-):
-    config_class = f"{kind}_docker_repository_config"
-    luigi.configuration.get_config().set(config_class, "tag_prefix", tag_prefix)
-    if docker_repository_name is not None:
-        luigi.configuration.get_config().set(
-            config_class, "repository_name", docker_repository_name
-        )
-    password_environment_variable_name = f"{kind.upper()}_DOCKER_PASSWORD"
-    if docker_username is not None:
-        luigi.configuration.get_config().set(config_class, "username", docker_username)
-        if docker_password is not None:
-            luigi.configuration.get_config().set(
-                config_class, "password", docker_password
-            )
-        elif password_environment_variable_name in os.environ:
-            print(
-                f"Using password from environment variable {password_environment_variable_name}"
-            )
-            password = os.environ[password_environment_variable_name]
-            luigi.configuration.get_config().set(config_class, "password", password)
-        else:
-            password = getpass.getpass(
-                f"{kind.capitalize()} Docker Registry Password for User %s:"
-                % docker_username
-            )
-            luigi.configuration.get_config().set(config_class, "password", password)
 
 
 def import_build_steps(flavor_path: Tuple[str, ...]):
+    # TODO Move to script-languages-container-tools: https://github.com/exasol/script-languages-container-tool/issues/268
```

Comment:
Was confused for the moment, because the framework for building the docker images is used also for the test container, but it is only about the build steps file.

Reply:
However, more correct would be probably extracting the docker images build part into a repo and the import with it

Reply:
Hm, I see that the `flavor_path` and  and `FlavorsBaseTask` and so on, which are tasks specific for exaslct are also here. So I guess you are right, and it's does not make sense to move this single function only then to exaslct. I will close https://github.com/exasol/script-languages-container-tool/issues/268

Reply:
However FlavorBaseTask doesn't belong here, contrary to the import build steps. Also the test env should not know about flavors, that should be added in slct